include/swift/AST/DiagnosticsSema.def

```diff
@@ -1499,7 +1499,7 @@ ERROR(access_control_extension_more,none,
       "be declared %select{%error|fileprivate|internal|public|%error}2",
       (AccessLevel, DescriptiveDeclKind, AccessLevel))
 ERROR(access_control_extension_open,none,
-      "extensions cannot use 'open' as their default access; use 'public'",
+      "extensions cannot be marked 'open'; mark individual members as 'open' instead",
```

Comment:
"Mark" works but I don't know that we use this term very often in Swift diagnostics. How about "be declared" as above and below?

Reply:
Seems good! Fixed 👍